clients/js/test/printV1.test.ts

```diff
@@ -0,0 +1,179 @@
+import { createMintWithAssociatedToken } from '@metaplex-foundation/mpl-toolbox';
+import { generateSigner, percentAmount, some } from '@metaplex-foundation/umi';
+import test from 'ava';
+import {
+  DigitalAsset,
+  DigitalAssetWithToken,
+  TokenStandard,
+  fetchDigitalAsset,
+  fetchDigitalAssetWithAssociatedToken,
+  findMasterEditionPda,
+  printSupply,
+  printV1,
+} from '../src';
+import { createDigitalAssetWithToken, createUmi } from './_setup';
+
+test('it can print a new edition from a NonFungible', async (t) => {
```

Comment:
Should we have a test for `ProgrammableNonFungible`?

Reply:
Oops you're right I forgot PNFTs. Tackling this now and marking the PR as draft in the meantime.

Reply:
Done ✅